package.json

```diff
@@ -47,6 +47,7 @@
     "react-custom-scrollbars": "4.2.1",
     "react-diff-viewer": "^1.1.0",
     "react-dom": "^16.4.0",
+    "react-draggable": "^3.3.0",
```

Comment:
Remove it.

Reply:
@akshatnitd I thought we could keep this. In future if we want to make it draggable we will need to just add one line. Shall I remove it still?

Reply:
Remove it